include/vk_mem_alloc_imported.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef VULKAN_MEMORY_ALLOCATOR_HPP
+#define VULKAN_MEMORY_ALLOCATOR_HPP
+#endif
+
+#ifndef VULKAN_MEMORY_ALLOCATOR_ENUMS_HPP
+#define VULKAN_MEMORY_ALLOCATOR_ENUMS_HPP
+#endif
+
+#ifndef VULKAN_MEMORY_ALLOCATOR_FUNCS_HPP
+#define VULKAN_MEMORY_ALLOCATOR_FUNCS_HPP
+#endif
+
+#ifndef VULKAN_MEMORY_ALLOCATOR_HANDLES_HPP
+#define VULKAN_MEMORY_ALLOCATOR_HANDLES_HPP
+#endif
+
+#ifndef VULKAN_MEMORY_ALLOCATOR_STRUCTS_HPP
+#define VULKAN_MEMORY_ALLOCATOR_STRUCTS_HPP
+#endif
```

Comment:
So it's a stub, which sets include guards, as if they were really included? Why do we need it and what's an intended usage for it? It's not used anywhere atm